index.html

```diff
@@ -5,145 +5,196 @@
             <ul class="boxes black">
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-            <button class="Project">See Project</button>
+            <button class="button">See Project</button>
         </article>
-        <br>
-        <article class="repeated">
-            <h2>Profesional Art Printing Data</h2>
+
+        <article class="repeated" id="six">
+            <h2 class="mobile">Profesional Art Printing Data</h2>
+            <h2 class="desktop">Website Protfolio </h2>
             <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
             <ul class="boxes black">
                 <li>html</li>
                 <li>bootstrap</li>
                 <li>Ruby</li>
             </ul>
-                <button class="Project">See Project</button>
+                <button class="button">See Project</button>
         </article>
+
     </section>
+
+<!-------------------Here starts ABOUT ME section-------------------------->
+
     <section class="aboutme">
         <h2>About me</h2>
         <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
-        <button>Get my resume</button>
-    </section>
-    <hr>
-    <section class="list">
-        <ul>
-            <li><img src="./Images/Rectangle55.png" alt="rectangle"> Language</li>
-            <li><ul>
-                <li><p class="listsub">Javascript</p></li>
-                <li><p class="listsub">Ruby</p></li>
-                <li><p class="listsub">Html</p></li>
-                <li><p class="listsub">Css</p></li>
-            </ul></li>
-
-            <li><img src="./Images/Rectangle56.png" alt="rectangle"> Frameworks</li>
-            <li><ul>
-                <li><p class="listsub">Bootstrap</p></li>
-                <li><p class="listsub">Ruby on Rails</p></li>
-                <li><p class="listsub">RSPec</p></li>
-                <li><p class="listsub">CapyBara</p></li>
-                <li><p class="listsub">Selenium</p></li>
-            </ul></li>
-
-            <li><img src="./Images/Circle.png" alt=""> Skills</li>
-            <li><ul>
-                <li><p class="listsub">Codekit</p></li>
-                <li><p class="listsub">Githubt</p></li>
-                <li><p class="listsub">Codepen</p></li>
-                <li><p class="listsub">Gitlab</p></li>
-                <li><p class="listsub">Terminal</p></li>
-            </ul></li>
-
-        </ul>
+        <button class="button">Get my resume</button>
     </section>
+    
+<!------------------Here is the LIST OF LANGUAGES--------------------------->
+        <section class="list">
```

Comment:
Sadly, the same issue remains in your design. Kindly make the necessary adjustments so that your design looks as close as possible to Figma's

![image](https://user-images.githubusercontent.com/15807343/149429001-8bc78bd7-494e-4f92-b751-751156579075.png)

Hint: Try changing the following property in your .aboutme class 

        height: 30vh;